plasmapy/formulary/collisions.py

```diff
@@ -2033,3 +2064,271 @@ def coupling_parameter(
+
+        Coulomb_log : `~astropy.units.Quantity`
+            The value of the Coulomb logarithm evaluated for the two interacting particles.
+
+        Raises
+        ------
+        `ValueError`
+            If both :math:`v_a` and :math:`T_a` are specified, or neither
+            :math:`v_a` nor :math:`T_a` are specified.
+
+        Notes
+        -----
+        The frequency of collisions between a test particle (subscript :math:`\alpha`) and
+        a field particle (subscript :math:`\beta`)  each with mass  :math:`m` and charge :math:`e`
+        are given by four differential equations:
+
+            momentum loss: :math:`\frac{d\textbf{v}_{α}}{dt}=-ν_{s}^{α\backslashβ}\textbf{v}_{α}`
+
+            transverse diffusion: :math:`\frac{d}{dt}\left(\textbf{v}_{α}-\overline{\textbf{v}}_{α}\right)_{⊥}^{2}=ν_{⊥}^{α\backslashβ}v_{α}^{2}`
+
+            parallel diffusion: :math:`\frac{d}{dt}\left(\textbf{v}_{α}-\overline{\textbf{v}}_{α}\right)_{∥}^{2}=ν_{∥}^{α\backslashβ}v_{α}^{2}`
+
+            energy loss: :math:`\frac{d}{dt}v_{α}^{2}=-ν_{ϵ}^{α\backslashβ}v_{α}^{2}`
+
+        These equations yield the exact formulas:
+
+            momentum loss: :math:`ν_{s}^{α\backslashβ}=\left(1+\frac{m_{α}}{m_{β}}\right)ψ\left(x^{α\backslashβ}\right)ν_{0}^{α\backslashβ}`
+
+            transverse diffusion: :math:`ν_{⊥}^{α\backslashβ}=2\left[\left(1-\frac{1}{2x^{α\backslashβ}}\right)ψ\left(x^{α\backslashβ}\right)\ +ψ'\left(x^{α\backslashβ}\right)\right]ν_{0}^{α\backslashβ}`
+
+            parallel diffusion: :math:`ν_{||}^{α\backslashβ}=\left[\frac{ψ\left(x^{α\backslashβ}\right)}{x^{α\backslashβ}}\right]ν_{0}^{α\backslashβ}`
+
+            energy loss: :math:`ν_{ϵ}^{α\backslashβ}=2\left[\left(\frac{m_{α}}{m_{β}}\right)ψ\left(x^{α\backslashβ}\right)-ψ'\left(x^{α\backslashβ}\right)\right]ν_{0}^{α\backslashβ}`
+
+        where,
+
+            :math:`ν_{0}^{α\backslashβ}=\frac{4\pi e_{α}^{2}e_{β}^{2}λ_{αβ}n_{β}}{m_{α}^{2}v_{α}^{3}}`,
+
+            :math:`x^{α\backslashβ}=\frac{m_{β}v_{α}^{2}}{2k_B T_{β}}`,
+
+            :math:`ψ\left(x\right)=\frac{2}{\sqrt{\pi}}\int_{0}^{x}t^{\frac{1}{2}}e^{-t}dt`,
+
+            :math:`ψ'\left(x\right)=\frac{dψ}{dx}`,
+
+        and :math:`\lambda_{\alpha \beta}` is the Coulomb logarithm for the collisions,
+        :math:`n_\beta` is the number density of the field particles, :math:`v_\alpha` is
+        the speed of the test particles relative to the field particles, :math:`k_B` is Boltzmann's
+        constant, and :math:`T_\beta` is the temperature of the field particles.
+
+        For values of x<<1 (the 'slow' or 'thermal' limit) or x>>1 (the 'fast' or 'beam' limit),
+        :math:`\psi` asymptotes to zero or one respectively. For simplified expressions in these limits
+        we encourage the curious reader to refer to p. 31 of :cite:t:`nrlformulary:2019`
+
+        Examples
+        --------
+        >>> import astropy.units as u
+        >>> T_a = 1 * u.eV
+        >>> n_b = 1e20 * u.cm**-3
```

Comment:
You should use MKS units in the examples for consistency with the rest of PlasmaPy

Reply:
Is expressing the temperature in electronvolts acceptable?

Reply:
That's fine (that's not really MKS or CGS, just plasma physicists being weird)